classes/class-wc-connect-taxjar-integration.php

```diff
@@ -933,10 +933,7 @@ public function calculate_tax( $options = array() ) {
 			}
 		}
 
-		// Remove taxes if they are set somehow and customer is exempt
-		if ( WC()->customer->is_vat_exempt() ) {
```

Comment:
This line is never reached because of the early return at https://github.com/Automattic/woocommerce-services/blob/develop/classes/class-wc-connect-taxjar-integration.php#L870